indexer/packages/postgres/src/stores/transfer-table.ts

```diff
@@ -431,6 +431,49 @@ export async function getNetTransfersPerSubaccount(
   return convertToSubaccountAssetMap(assetsPerSubaccount);
 }
 
+export async function getNetTransfersBetweenSubaccountIds(
+  sourceSubaccountId: string,
+  recipientSubaccountId: string,
+  assetId: string,
+  options: Options = DEFAULT_POSTGRES_OPTIONS,
+): Promise<string> {
+  const queryString: string = `
+  SELECT 
+    SUM(sub."size") AS "totalSize"
```

Comment:
non-blocking nit: You can coalesce with 0 to make sure this always returns a number I think.